server/graphql/v2/mutation/EmojiReactionMutations.js

```diff
@@ -0,0 +1,160 @@
+import { GraphQLNonNull, GraphQLObjectType, GraphQLString } from 'graphql';
+
+import { mustBeLoggedInTo } from '../../../lib/auth';
+import models from '../../../models';
+import { canComment } from '../../common/expenses';
+import { Forbidden, NotFound, Unauthorized, ValidationFailed } from '../../errors';
+import { idDecode, IDENTIFIER_TYPES } from '../identifiers';
+import { CommentReferenceInput } from '../input/CommentReferenceInput';
+import { UpdateReferenceInput } from '../input/UpdateReferenceInput';
+import { Comment } from '../object/Comment';
+import Update from '../object/Update';
+
+/**
+ * Object type for EmojiReaction mutation.
+ */
+const EmojiReactionsResponse = new GraphQLObjectType({
+  name: 'EmojiReactionOutput',
```

Comment:
The names must match here

```suggestion
const EmojiReactionsResponse = new GraphQLObjectType({
  name: 'EmojiReactionResponse',
```

Reply:
Done. 👍🏽